src/ripple/overlay/README.md

```diff
@@ -344,7 +344,7 @@ messages for the local and remote endpoints, and combine them to generate a uniq
 endpoints.
 
 That fingerprint, which is never shared over the wire (since each endpoint will
-calculate it independently), is then signed by each server using its public
```

Comment:
I believe this refers to the fact that the server's identity is public. It is not a reference to a public key.

I can see how the wording is confusing, however. Let me see if I can propose a larger rewrite of the paragraph.

Reply:
Suggestion:

> That fingerprint is calculated by each endpoint independently, so the fingerprint is never transmitted over the network. Each server then utilizes its private key to sign the fingerprint. This is the same keypair that determines the server's public `secp256k1` node identity. The signature is transferred over the secure SSL/TLS encrypted link during the protocol's initial handshake phase.

Reply:
thanks!

Reply:
Do you know if I can incorporate your suggestion automatically as a new commit? I can copy the text, but I vaguely remember that there is a way to do it much more easily via the github UI

Reply:
There is a way to make apply-able suggestions in the GitHub UI, but it doesn't work for modifying deleted lines. I went ahead and added my suggestion as a new commit. 👍